plugins/modules/swim_workflow_manager.py

```diff
@@ -1494,17 +1496,15 @@ def get_diff_activation(self):
             return self
 
         if len(device_uuid_list) == 0:
-            self.status = "failed"
-            self.msg = "No devices found for Image Activation"
+            self.status = "success"
+            self.msg = "No matched device(s) for swim image activation task."
```

Comment:
self.msg = "The SWIM image activation task could not proceed because no eligible devices were found."

Reply:
updated.